### ISSUE TITLE
Allow import aws_ssm_patch_baseline

### ISSUE THREAD

**Issue by nunofernandes** [terraform-providers/terraform-provider-aws#7816]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

Allow the importing of an existing aws_ssm_patch_baseline

### New or Affected Resource(s)

* aws_ssm_patch_baseline

### Potential Terraform Configuration

```
terraform import aws_ssm_patch_baseline.mypatchline pb-XXXXXXX
```

**Comment by bflad** [comment]
Support for this has been merged and will release with version 2.2.0 of the Terraform AWS Provider, likely later today.

**Comment by bflad** [comment]
This has been released in [version 2.2.0 of the Terraform AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#220-march-15-2019). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.
